src/Microsoft.DotNet.Cli.Utils/Telemetry.cs

```diff
@@ -0,0 +1,99 @@
+using System;
+using System.Collections.Generic;
+using Microsoft.ApplicationInsights;
+using Microsoft.Extensions.PlatformAbstractions;
+
+namespace Microsoft.DotNet.Cli.Utils
+{
+    public class Telemetry
+    {
+        private class Variables
+        {
+            public static readonly string InstrumentationKey = "74cc1c9e-3e6e-4d05-b3fc-dde9101d0254";
+            private static readonly string Prefix = "DOTNET_CLI_TELEMETRY_";
+            public static readonly string Optout = Prefix + "OPTOUT";
+        }
+
+        private class Properties
+        {
+            public static readonly string OSVersion = "OS Version";
+            public static readonly string OSPlatform = "OS Platform";
+            public static readonly string RuntimeId = "Runtime Id";
+            public static readonly string ProductVersion = "Product Version";
+        }
+
+        private static bool _isInitialized = false;
+        private static TelemetryClient _client = null;
+
+        private static Dictionary<string, string> _commonProperties = null;
```

Comment:
Static variables should be "s_" prefixed.  "_" prefix is for instance variables.